src/GeositeFramework/js/Map.js

```diff
@@ -307,8 +315,13 @@ require(['use!Geosite',
                 return !layer.graphics && layer.visible;
             });
 
-            // Subtract one for the base layer.
-            if ((totalVisibleLayers.length - 1) !== nuggetCount) {
+            // Subtract one for the base layer and one for each 
+            // custom legend that is visible and has children
```

Comment:
I'll rebase a change to this comment that includes the description of the `serviceOptOut` after review.